clients/fides-js/src/fides.ts

```diff
@@ -133,7 +134,7 @@ const automaticallyApplyGPCPreferences = (
   fidesRegionString: string | null,
   fidesApiUrl: string,
   debug: boolean,
-  effectiveExperience?: PrivacyExperience | null
+  effectiveExperience?: PrivacyExperience
```

Comment:
hmmmm to be really TS correct this should probably be `PrivacyExperience | {}`. I suspect that could cause other errors, but I think it'd be worth it since `{}` is kind of a tricky case (it's not falsy)

Reply:
ah so I call the `experienceHasNotices` method before calling `automaticallyApplyGPCPreferences`, which will check for `Object.keys(experience).length >= 0`. Unless you think I should move that check internal to `automaticallyApplyGPCPreferences` instead?